Tensile/Source/lib/include/Tensile/ContractionProblem.hpp

```diff
@@ -35,17 +36,17 @@
 namespace Tensile
 {
     /**
-     * \addtogroup Problem
-     * @{
-     */
+ * \addtogroup Problem
+ * @{
+ */
 
     /**
-     * Describes a tensor contraction in by using TensorDescriptor objects for
-     * each input or output tensor as well as indices describing transposes,
-     * summations, etc. This is decoupled from any particular pointers, which
-     * are provided in ContractionInputs objects.
-     */
-    class TENSILE_API ContractionProblem: public Problem
+ * Describes a tensor contraction in by using TensorDescriptor objects for
```

Comment:
Is there a setting which preserves the indentation of these comments?  This is pretty ugly now.

Reply:
This is what I'm talking about.  The indentation of lines 2+ of the comments have not been preserved.

Reply:
This change has already been merged into multiple other branches. Reverting it will be as disruptive, if not more. I am fully willing to help with merging, as I have spent extra time writing the instructions on https://github.com/ROCmSoftwarePlatform/Tensile/pull/982#issue-419544566

Three others have already merged my changes into their branches.

I ran a build of rocBLAS with this change, and it had no issues. But I realize that there are other clients.

We can refine the reformatting, which is why I asked for feedback (and also especially for Python).

In the meantime, if there are drastic formatting differences, then I agree with Saad: Use `// clang-format off` and `// clang-format on` to delimit where reformatting should not be performed.

Reply:
I'm not arguing for reverting this PR.  I'm asking if we can tweak the settings so we don't have to manually disable formatting in each of the 130 locations where this style of comment exists in Tensile.

Reply:
> 
> 
> I'm not arguing for reverting this PR. I'm asking if we can tweak the settings so we don't have to manually disable formatting in each of the 130 locations where this style of comment exists in Tensile.

I'll need to investigate `clang-format` options here, but maybe @saadrahim or @amdkila can answer more quickly.